src/lib/rest/rest.cpp

```diff
@@ -1320,10 +1320,29 @@ static int connectionTreat
       lmTransactionSetFrom(ip);
     }
 
-    char tenant[SERVICE_NAME_MAX_LEN + 1];
+    //
+    // Max name length of a database name for a tenant:
+    // Remember the database name is created like this:
+    //   orion-<tenant>
+    //
+    // 'orion' is the default database name prefix, but this
+    // string is configurable via CLI.
+    // However, its maximum allowed length is SERVICE_NAME_MAX_LEN, so
+    // this we can always use.
+    //
+    // Now:
+    //   DB_NAME_MAX_LEN + 1 + SERVICE_NAME_MAX_LEN + 1:
+    //     DB_NAME_MAX_LEN:      'orion'
+    //     1:                    '-'
+    //     SERVICE_NAME_MAX_LEN: tenant name, via HTTP header 'Fiware-Service' 
+    //     1:                    zero-termination of the string
+    // 
+    //
+    char tenant[DB_NAME_MAX_LEN + 1 + SERVICE_NAME_MAX_LEN + 1];
+
     ciP->tenantFromHttpHeader = strToLower(tenant, ciP->httpHeaders.tenant.c_str(), sizeof(tenant));
+    ciP->outMimeType          = mimeTypeSelect(ciP, "Normal");
```

Comment:
Not a clue, I just indented the line :-)

Looked it up:

```
static MimeType mimeTypeSelect(ConnectionInfo* ciP, const std::string& what)
{
  if ((what == "Error") ||   // JSON Error to be returned
      (what == "Normal"))    // Normal handling - same as Error
  {
    if (ciP->httpHeaders.accepted("application/json"))
    {
      return JSON;
    }
    if (ciP->httpHeaders.accepted("text/plain"))
    {
      return TEXT;
    }

    return JSON;
  }

  return JSON;
}
```

As it seems, it is not used. "Error" and "Normal" are treated the same way.

I can fix that entire thing inside this PR, if you like